File: src/main/java/no/nav/syfo/domain/dto/SoknadPeriode.java
package no.nav.syfo.domain.dto;

import lombok.Builder;
import lombok.Getter;
import lombok.Value;

import java.time.LocalDate;

@Value
@Builder
@Getter
public class SoknadPeriode {

    LocalDate fom;
    LocalDate tom;
    Integer grad;
}

File: src/main/kotlin/no/nav/helse/flex/domain/dto/SoknadPeriode.kt
package no.nav.helse.flex.domain.dto

import java.time.LocalDate

data class SoknadPeriode(
    val fom: LocalDate? = null,
    val tom: LocalDate? = null,
    val grad: Int? = null,
    val faktiskGrad: Int? = null,
    val sykmeldingstype: String? = null
)
